src/include/migraphx/check_shapes.hpp

```diff
@@ -332,7 +332,7 @@ struct check_shapes
      */
     const check_shapes& not_broadcasted() const
     {
-        if(not this->all_of([](const shape& s) { return not s.broadcasted(); }))
+        if(not this->all_of([](const shape& s) { return s.standard() or not s.broadcasted(); }))
```

Comment:
I think this change is fine.

Reply:
That solved some errors, but in some of the perf tests I'm now seeing: `RuntimeError: /src/AMDMIGraphX/src/targets/gpu/gemm_impl.cpp:85: blas_shape: GPU_GEMM: matrix dimensions can't be broadcasted` with this change.  Not sure how to properly correct that check.

Reply:
What test does that fail on?

Reply:
I wonder why we are not inserting a contiguous here.

Reply:
> What test does that fail on?

You can see them by going to the Conversation tab of this PR and looking for the big red X icons in the perf. test results.  All of the tests that threw errors had this same error.  They are:
```
torchvision-resnet50_1
cadene-resnext101_1
resnet50v1_fp16
resnet50v1_int8
```

Reply:
> I wonder why we are not inserting a contiguous here.

In `check_shapes`?  That doesn't seem a very appropriate place to do it.

Reply:
No, contiguous is inserted in auto_contiguous for when the shapes are not standard(or not the same as the normalize_standard shape). Then after lowering in eliminate_contiguous, we try to remove the contiguous, but only if it doesnt throw an error like this one, which means there should be a contiguous inserted to prevent this error, but somehow it escaped. 

We should try to get a backtrace to see where this happens, or run it with MIGRAPHX_TRACE_COMPILE=1 to see which pass it fails on.

Reply:
I've been trying to replicate the failures on `aus-navi3x-04.amd.com` and can't--commands such as `MIGRAPHX_TRACE_COMPILE=1 bin/driver verify --onnx  ../../resnext101_64x4di1.onnx` for instance, handle the same model files just fine.  I can't troubleshoot the location of the problem until I can replicate it.  Any ideas on how to track down what's different?

Reply:
But what about the failures for resnet50 models? Can you reproduce those?

Also, these are a compile failure so you just do `./bin/driver compile` instead of verify(that should be much quicker).

Reply:
> No, I just tried those again, and they don't fail either.

Ok, I think you need to try on MI series. The navi always uses MLIR which mean rocblas is never used and this error comes from rocblas.

> if I'm reading the aliasing in the script files correctly--tests with resnet50v1_fp16 and resnet50v1_int8 both use the same model, resnet50/resnet50_v1.onnx but with command line arguments --fp16 and --int8 respectively.

Yes.